### ISSUE TITLE
Problem with Credentials

### ISSUE THREAD

**Issue by gabrielfior** [mopidy/pyspotify#189]
Hello,

I am trying to play a track from Spotify using pyspotify in a Raspberry Pi, but I have problems with the credentials.
I downloaded the Spotify key from my developer account (Binary) and stored it in the same folder as the file. 
I am trying to run the example play_track.py. 
It would be great if someone could help me with this issue.

Below my error: 

```
pi@raspberrypi:~/spoti $ python play_track.py 
Traceback (most recent call last):
  File "play_track.py", line 61, in <module>
    session.relogin()
  File "/usr/lib/python2.7/dist-packages/spotify/session.py", line 190, in relogin
    spotify.Error.maybe_raise(lib.sp_session_relogin(self._sp_session))
  File "/usr/lib/python2.7/dist-packages/spotify/error.py", line 31, in maybe_raise
    raise LibError(error_type)
spotify.error.LibError: No credentials stored
```

The example file (just for completion).
```
#!/usr/bin/env python


#This is an example of playing music from Spotify using pyspotify.
from __future__ import unicode_literals

import sys
import threading

import spotify

if sys.argv[1:]:
    track_uri = sys.argv[1]
else:
    track_uri = 'spotify:track:6xZtSE6xaBxmRozKA0F6TA'

# Assuming a spotify_appkey.key in the current dir
session = spotify.Session()

# Process events in the background
loop = spotify.EventLoop(session)
loop.start()

# Connect an audio sink
audio = spotify.AlsaSink(session)

# Events for coordination
logged_in = threading.Event()
end_of_track = threading.Event()


def on_connection_state_updated(session):
    if session.connection.state is spotify.ConnectionState.LOGGED_IN:
        logged_in.set()


def on_end_of_track(self):
    end_of_track.set()


# Register event listeners
session.on(
    spotify.SessionEvent.CONNECTION_STATE_UPDATED, on_connection_state_updated)
session.on(spotify.SessionEvent.END_OF_TRACK, on_end_of_track)

# Assuming a previous login with remember_me=True and a proper logout
session.relogin()
#session.login()

logged_in.wait()

# Play a track
track = session.get_track(track_uri).load()
session.player.load(track)
session.player.play()

# Wait for playback to complete or Ctrl+C
try:
    while not end_of_track.wait(0.1):
        pass
except KeyboardInterrupt:
    pass
```
Thanks a lot,
Gabriel.

**Comment by trygveaa** [comment]
To use `session.relogin()` you will have to have run a script that logs in using your credentials and then logs out properly beforehand, so your credentials are stored. Have you done that?

I modified your script by replacing `session.relogin()` with
```
username = raw_input('Username: ')
password = raw_input('Password: ')
session.login(username, password, remember_me=True)
```
And added `session.logout()` to the end of it.

Then I ran the script, logged in, waited for the music to start, and pressed ctrl-c. After that I changed the script back to what it is in your post and tried to run it. Then it worked fine for me.
